src/TimebarWrapper.tsx

```diff
@@ -78,6 +89,13 @@ const TimebarWrapper = (props: any) => {
                   graphTracks={graph}
                 />
               )}
+              {highlightedTime && (
+                <TimebarHighlighter
+                  {...props}
```

Comment:
I'd like to avoid this use case because of:
- we don't know which props are needed
- performance impact
so have you identified which props do we need ? if we need all of them could be ok but probably not, what are your thoughts ?